kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -34,6 +32,10 @@
 #       }
 # }
 
+# Class that manages Kedro's library components.
+# from kedro.framework.context import KedroContext
+# CONTEXT_CLASS = KedroContext
```

Comment:
I move context down because I think `DataCatalog` and `KedroContext` are unlikely to be changed.